.gitignore

```diff
@@ -179,3 +179,8 @@ $RECYCLE.BIN/
 
 # Visual Studio Code
 .vscode/
+
+# JetBrains Rider
+.idea/
+*.DotSettings
```

Comment:
You're right, it does belong to ReSharper since R# in VS can also produce those files.